config/configauth/authenticator.go

```diff
@@ -31,14 +31,11 @@ var (
 
 // Authenticator will authenticate the incoming request/RPC
 type Authenticator interface {
-	io.Closer
+	component.Exporter
```

Comment:
Should this be component.Extension instead?

Reply:
Yes! Good catch.